suite-common/thp/src/connectThpDeviceThunk.ts

```diff
@@ -31,9 +31,13 @@ export const connectThpDeviceThunk = createThunk<void, ConnectThpDeviceThinkPara
                 dispatch(thpActions.incrementCredentialConnectionCounter({ credential }));
             }
 
+            const hasAutoconnectCredential =
+                (device?.thp?.credentials ?? []).find(it => it?.autoconnect) !== undefined;
```

Comment:
Nit: `?.find` simplifies
```suggestion
                device?.thp?.credentials?.find(it => it?.autoconnect) !== undefined;
```